src/nodes/NodeConnectionManager.ts

```diff
@@ -1074,11 +1096,11 @@ class NodeConnectionManager {
    * @param port port of the target client.
    * @param ctx
    */
-  public async holePunchReverse(
+  public holePunchReverse(
     host: Host,
     port: Port,
-    ctx?: Partial<ContextTimed>,
-  ): Promise<void>;
+    ctx?: Partial<ContextTimedInput>,
+  ): PromiseCancellable<void>;
   @ready(new nodesErrors.ErrorNodeConnectionManagerNotRunning())
   @timedCancellable(
     true,
```

Comment:
If there's no "forward" version of this, there should not be a "reverse" version of this.

Reply:
I'll rename it to `holePunch`.